eng/Version.Details.xml

```diff
@@ -181,6 +181,11 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>aafa91036e1efd3d4dcb67eeb261cb6d8f774685</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-externals" Version="7.0.0-alpha.1.22369.2">
```

Comment:
The location of this dependency seems arbitrary.  I would suggest moving it so that all dependencies from dotnet/runtime are grouped.

Reply:
The elements under `ProductDependencies` are sorted by dependency name rather than being grouped by origin repo.